minecraft_servers/iostein/manifest.json

```diff
@@ -9,7 +9,7 @@
   "social": {
     "web": "https://iostein.net",
     "web_shop": "https://iostein.net/credits",
-    "web_support": "https://forum.iostein.net",
+    "web_support": "https://iostein.net/discord",
```

Comment:
Web support cannot be just link to Discord/forum; add FAQ or contact form (if non-existent, remove)